### PR TITLE
[MIG][11.0] hr_phone_validation

### PR DESCRIPTION
**Pull request by fmdl**
from https://github.com/OCA/hr/pull/423

### CODE REVIEW

**Comment by fmdl** [comment]
Hi guys I was made the PR from a wrong branch.

@feketemihai
@simahawk

Thanks

File: hr_phone_validation/README.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (hieulucky111) [inline hr_phone_validation/README.rst]: Please follow the template of OCA: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: hr_phone_validation/tests/test_phone_validation.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestHrPhoneValidation(TransactionCase):
+
+    def test_phone_validation(self):
+        Employee = self.env['hr.employee']
```
Comment (hieulucky111) [inline hr_phone_validation/tests/test_phone_validation.py]: Variable name should be lowercase as `hr_employee`: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#variable-names

**Review by hieulucky111 — APPROVED** [review]
some minor adjustments, LGTM over all.

File: hr_phone_validation/tests/test_phone_validation.py (hunk 2 above)
Comment (denislour) [reply hr_phone_validation/tests/test_phone_validation.py]: In this case, you should change variable `Employee` into `hr_employee_env` to indicate it's environment variable.

**Review by denislour — CHANGES_REQUESTED** [review]

**Review by feketemihai — APPROVED** [review]

File: hr_phone_validation/tests/test_phone_validation.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestHrPhoneValidation(TransactionCase):
+
+    def test_phone_validation(self):
+        employee = self.env['hr.employee'].search([], limit=1)
+        phone = '+33612345678'
+        employee.write({'work_phone': phone, 'mobile_phone': phone})
+        employee._onchange_work_phone_validation()
```
Comment (simahawk) [inline hr_phone_validation/tests/test_phone_validation.py:10]: @fmdl nothing is tested here :smile:

**Review by simahawk — CHANGES_REQUESTED** [review]

File: hr_phone_validation/tests/test_phone_validation.py (hunk 3 above)
Comment (fmdl) [reply hr_phone_validation/tests/test_phone_validation.py:10]: @simahawk I don't know why on test is OK and an other is not OK.

Comment (simahawk) [reply hr_phone_validation/tests/test_phone_validation.py:10]: @fmdl can you elaborate? :)

Comment (fmdl) [reply hr_phone_validation/tests/test_phone_validation.py:10]: image

Comment (simahawk) [reply hr_phone_validation/tests/test_phone_validation.py:10]: @fmdl sorry, missed that. Hmm the test seems to tell us the value has changed, is not equal anymore to what we expected: any other onchange/override/whatever that might change the value?
You should run the test locally precisely as travis does in that scenario to see what happens.

Comment (astirpe) [reply hr_phone_validation/tests/test_phone_validation.py:10]: Probably this test failure can be fixed by forcing the employee to have his own country_id.
Could you give it a try?

**Review by astirpe — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
Is this going to be fixed?

**Comment by jarroyomorales** [comment]
@fmdl I did a PR to your branch fixing travis. Can you check and merge?
https://github.com/fmdl/hr/pull/2

**Review by jarroyomorales — APPROVED** [review]
LGTM! Could you squash the merge commit?

@dreispt @simahawk Can you check again? Thank you!

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
> LGTM! Could you squash the merge commit?

+1

**Pull request closed without merging**